allure-pytest-bdd/src/utils.py

```diff
@@ -50,5 +44,5 @@ def get_pytest_report_status(pytest_report):
 
 def get_params(node):
     if hasattr(node, 'callspec'):
-        params = node.callspec.params
+        params = node.callspec.params.pop('_pytest_bdd_example', {})
```

Comment:
Let's save the original params, other than `_pytest_bdd_example`, as well. Because, well, I know, it looks creepy, but it is possible to combine outline parameters with `@pytest.mark.parametrize`:

```python
@pytest.mark.parametrize("a", [1, 2])
@scenario("issue636.feature", "Reproducing the issue 636")
def test_reproduction(a):
    pass
```

Also, I think it's a better practice, if possible, not to modify the input variable (namely, `node.callspec.params`).

```suggestion
        params = dict(node.callspec.params)
        outline_params = params.pop('_pytest_bdd_example', {})
        params.update(outline_params)
```